File: app/src/main/java/com/example/msg91/ApplicationClass.java
package com.example.msg91;

import android.app.Application;

import com.msg91.sendotpandroid.library.internal.SendOTP;

public class ApplicationClass extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        SendOTP.initializeApp(this,"343141A0eUofjHNg5f73eff8P1");        //initialization
    }
}

File: app/src/main/java/com/example/kotlinmsg91/ApplicationClass.kt
package com.example.kotlinmsg91

import android.app.Application
import com.msg91.sendotpandroid.library.internal.SendOTP

class ApplicationClass : Application() {
    override fun onCreate() {
        super.onCreate()
        SendOTP.initializeApp(this, "343141A0eUofjHNg5f73eff8P1")
    }
}